docs/conf.py

```diff
@@ -25,7 +25,7 @@
 
 source_suffix = {".rst": "restructuredtext"}
 templates_path = ["_templates"]
-html_static_path = ["assets"]
+html_static_path = ["assets", "_static"]
```

Comment:
icon images could not be imported in `raw html` directive somehow when images are in `assets` folder. It works `_static`.

Reply:
that's great!

Reply:
This happened to me, too. I ended up adding the `_static` to `html_static_path` just like what you did.